### PR TITLE
Changing report name to zabapgit_full in merge

### PR DESCRIPTION
**Pull request by valdirmendesgt**
Implements this feature: [https://github.com/larshp/abapGit/issues/2370](https://github.com/larshp/abapGit/issues/2370)

### CODE REVIEW

**Comment by valdirmendesgt** [comment]
Before merge this PR, we need upgrade the abapmerge version to 0.13 (https://github.com/larshp/abapmerge/issues/95) because this version has CLI option to change the report name in merge process. Do we need create an issue to upgrade abapmerge version?

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "license": "MIT",
   "scripts": {
     "test": "npm run eslint && npm run abaplint",
-    "merge": "abapmerge -f src/zabapgit.prog.abap > zabapgit.abap",
+    "merge": "abapmerge -f src/zabapgit.prog.abap > zabapgit.abap -c zabapgit_full",
```
Comment (sbcgua) [inline package.json]: did you mean `abapmerge -f src/zabapgit.prog.abap -c zabapgit_full > zabapgit.abap` ?

Comment (valdirmendesgt) [reply package.json]: Yes, you're right.

Comment (valdirmendesgt) [reply package.json]: I fixed.

**Comment by larshp** [comment]
upgrading, just change the following line in package.json
`"abapmerge": "^0.11.5",`

**Comment by valdirmendesgt** [comment]
Thanks @larshp. I will change now.

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
   },
   "devDependencies": {
     "abaplint": "^2.1.6",
-    "abapmerge": "^0.11.5",
+    "abapmerge": "^0.13",
```
Comment (larshp) [inline package.json]:
```suggestion
    "abapmerge": "^0.13.0",
```

**Comment by larshp** [comment]
looks like something is missing somewhere, https://github.com/abapGit/build/commit/013297ca506f847670bfa43634d21a69b43612a9

**Comment by valdirmendesgt** [comment]
@larshp I think the error is in the parseArgs method in abapmerge cli. I will write tests to ensure this.

**Pull request merged**